Packages/com.unity.inputsystem/Documentation~/Actions.md

```diff
@@ -18,10 +18,13 @@ You can then get a reference to this action in your code, and check its value, o
 
 Actions also make it simpler to create a system that lets your players [customize their bindings at runtime](ActionBindings.md#interactive-rebinding), which is a common requirement for games.
 
->**Note**
->Actions are a runtime only feature. You can't use them in `EditorWindow` code.
-
->__Note__: It is possible to read input without using Actions and Bindings by directly reading specific device controls. This is less flexible, but can be quicker to implement for certain situations. Read more about [directly reading devices from script](Workflow-Direct.md).
+>**Notes:**
+> - Actions are a runtime only feature. You can't use them in [Editor window code](xref:EditorWindow).
```

Comment:
An error is reported in the test run about this line
https://unity-ci.cds.internal.unity3d.com/job/33610680/results

Message: Expected: <empty> But was: < "xref:EditorWindow in C:\build\output\Unity-Technologies\InputSystem\Temp\docstest\com.unity.inputsystem@1.8\manual\Actions.html (File Not Found)" > StackTrace: at DocumentationBasedAPIVerficationTests.API_DocumentationManualDoesNotHaveMissingInternalLinks () [0x0004a] in C:\build\output\Unity-Technologies\InputSystem\Assets\Tests\InputSystem\DocumentationBasedAPIVerficationTests.cs:90 Test